includes.php

```diff
@@ -289,7 +289,7 @@ function make_shell_command( array $env, string $cmd ): string {
 function shell_echo( string $cmd ): int {
 	echo '<pre>';
 	echo htmlspecialchars( "$cmd\n" );
-	$process = Process::fromShellCommandline( $cmd );
+	$process = Process::fromShellCommandline( $cmd, null, [] );
```

Comment:
We should probably use this $env argument instead of building the command ourselves with make_shell_command.

Reply:
See #377

Reply:
Make sense, the current code is from before this used `Process::fromShellCommandline()`.